### ISSUE TITLE
Cannot require('card') with browserify

### ISSUE THREAD

**Issue by kavun** [jessepollak/card#190]
Trying to use card with browserify. I'm getting an error

    Error: Cannot find module './lib/extend' from 'C:\Projects\card-test\node_modules\card\lib\js'

My package.json

```js
{
  "name": "card-test",
  "version": "1.0.0",
  "description": "card browserify test",
  "main": "index.js",
  "scripts": {
    "build": "browserify . > bundle.js"
  },
  "devDependencies": {
    "browserify": "^11.0.0"
  },
  "dependencies": {
    "card": "^1.1.0"
  }
}
```

My index.js

    var card = require('card');

Running `npm run build` gives me the error above.

Any thoughts?

**Comment by chrisworrell** [comment]
I am running into the exact same error.  Looks like the card.js is not referencing lib/extend properly since the extend.js resides in the node_modules and not where it is attempting to call it from in card.js unless of course I am missing some sort of setup step.

Really would love some help here as I really like the demo and would love to implement

**Comment by theopolisme** [comment]
This is indeed a drag, and hopefully can be fixed asap (this is a known problem with Browserify and attempting to require() browserified modules, see https://github.com/substack/node-browserify/pull/1151).  One fix is indeed minifying the source files (see #191).

As a temporary fix I'm using card@1.0.0, which doesn't have this problem.

Thanks for the great library!

**Comment by duluca** [comment]
Related to https://github.com/jessepollak/card/issues/197

**Comment by graup** [comment]
Can confirm card@1.0.0 works, will also use that version until this is fixed. Thanks all!

**Comment by jessepollak** [comment]
going to look into this — does anyone have a minimal repo that can show the error?

**Comment by jessepollak** [comment]
Nevermind, got it! Fix will go out in `v1.2.2`.
